freqtrade/edge/edge_positioning.py

```diff
@@ -238,20 +238,13 @@ def _fill_calculable_fields(self, result: DataFrame) -> DataFrame:
         :param result Dataframe
         :return: result Dataframe
         """
-
-        # stake and fees
-        # stake = 0.015
-        # 0.05% is 0.0005
-        # fee = 0.001
-
-        # we set stake amount to an arbitrary amount.
-        # as it doesn't change the calculation.
-        # all returned values are relative.
-        # they are defined as ratios.
+        # We set stake amount to an arbitrary amount, as it doesn't change the calculation.
+        # All returned values are relative, they are defined as ratios.
         stake = 0.015
-        fee = self.fee
-        open_fee = fee / 2
-        close_fee = fee / 2
+
+        # Fee should be applied twice
+        open_fee = self.fee
+        close_fee = self.fee
```

Comment:
The assignment of this to two seperate variables seems unnecesary.

I know it's just pointers - but they're not changed (and only ever used once after this line), so why not use `self.fee` below as well (or at least assign it to `fee = self.fee`)?

I do remember that i did notice this "halving" problem before - but i don't remember why it wasn't fixed then. 

@mishaker - i think i discussed this with you - hope you remember (?)

Reply:
I think it was "reserved" for the future, when different fees for close and open can be used...

Reply:
but for backtesting / edge - that's never really applicable as we lack the info for that.

Reply:
well, theoretically, there can be `--close-fee` and `--open-fee`, instead of explaining that `--fee` is a cumulative for both, which can be misunderstooden...

Reply:
That's right. it is not necessary. using self.fee is enough.

Reply:
Done